ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/management-security.yml

```diff
@@ -5,3 +5,12 @@ management_security:
     minimum_length: 17
     encryption_key_common: true
     encryption_reversible: aes-256-gcm
+  ssl_profiles:
+    - name: tls-versions-profile
+      tls_versions: "1.0 1.1"
+    - name: cipher-list-profile
+      cipher_list: ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384
```

Comment:
Wouldn't it make more sense to use a list in the data model and then create the colon separated list in the template?
so
```yaml
ssl_profiles:
  - name: xxx
    ciphers:
      - ECDHE-RSA-AES256-GCM-SHA384
      - ECDHE-RSA-AES256-SHA384
```
In the docs, we should then print as comma separated to make it more readable.
Or is following OpenSSL's format better?

Reply:
thought about this as well - basically the only thing that is important is the order - cf: https://www.mkssoftware.com/docs/man1/openssl_ciphers.1.asp so we could have a list instead.

List may be easier to read though in some occasions it could be confusing:

e.g. for `ALL:!COMPLEMENTOFDEFAULT:!eNULL.`

we would end up with
```
ciphers:
  - ALL
  - !COMPLEMENTOFDEFAULT
  - !eNULL
```

Which is kind of weird when reading as it looks like we include all ciphers.

Reply:
IMO those two are equally confusing :)

Reply:
haha well I don't have a string feeling for any - might be simpler for people to use the string if they already have it but happy to go with the list version as well. There may be things I don't foresee doing this as I have not played that much with the syntax but we can always change at a later stage if need be. 

The only thing is that each new element of the list impact the "selected" list of ciphers so the order is paramount.
As long as we keep the order of the list in the template it stays usable.

Reply:
Just skimmed the examples from openssl, and since it can have characters like `= + -` etc I think we should just stick with a single string like you have. Sorry for the noise :)